src/Restaurant.js

```diff
@@ -1,17 +1,15 @@
-import React from "react";
+import Order from "./Order";
 
 const Restaurant = () => {
-  const pizzas = 0;
   return (
     <div>
       <h3>Restaurant Orders</h3>
-      
-        <ul className="list">
-          <Order orderType="Pizza" />
-          <Order orderType="Salads" />
-          <Order orderType="Chocolate cake" />
-        </ul>
-      
+      <ul className="order-type">
+        <Order orderType="Pizza" />
+        <Order orderType="Cesar-Salad" />
+        <Order orderType="Chocolate cake" />
+        <Order orderType="Ice-Cream"/>
```

Comment:
You could instead create an array of order types `const orders = ["Pizza", "Cesar sald", "Ice cream"]` and `{orders.map(order => <Order orderType={order} />)}` to save some repetition